### PR TITLE
test PR 

### PR DESCRIPTION
**Pull request by vojtechhuser**
minimum changes done (partial fix for #162)
(testing the December 2016 path to travisCI testing)

### CODE REVIEW

**Comment by chrisknoll** [comment]
Please give this PR a suitable name to indicate the topic of changes being applied in this PR.

**Comment by vojtechhuser** [comment]
this was a test request. I was hoping to test if travisCI would test my code. It only works (as advertised) on internal branches :-(

**Comment by chrisknoll** [comment]
Travis won't test this branch, it needs to be put into a branch local to the OHDSI repo (this branch is in your fork).

Also get into a habit of creating branches in your local repo for changes. Don't apply commits right to the master branch because once this gets committed into the OHDSI repo you'll want to refresh the OHDSI repo master branch on top of your own forked' master branch.  And if you make your own commits to your forked master, you'll have trouble trying to merge the commits from the OHDSI master into your own master.

In short, don't commit anything to your local master branch, only get those commits from the OHDSI's master branch.

**Comment by vojtechhuser** [comment]
Well, this will be even more learning. I took me a while to learn the branches back in early 2016 and now even branches in the forked repo. Oh boy. I wish the old way (inside a branch of the main project) and a convention to commit to generation of minimum "commit noise" would be an option for me. :-( @chrisknoll

**Comment by chrisknoll** [comment]
@vojtechhuser : you have internal collaborator rights I believe to this repo.  If you can follow the process to create specific feature branches off the OHDSI repo and create pull requests off of these feature branches, then travis CI should be able to execute the test cases for you.
